src/components/Modals/DeployModal/DeployToLand/LandAtlas/LandAtlas.tsx

```diff
@@ -25,8 +25,10 @@ export default class LandAtlas extends React.PureComponent<Props, State> {
 
   componentDidMount() {
     this.mounted = true
-    const { onFetchDeployments } = this.props
-    onFetchDeployments()
+    const { isLoggedIn, onFetchDeployments } = this.props
+    if (isLoggedIn) {
+      onFetchDeployments()
```

Comment:
this now only happens when logged in?

Reply:
yes, if you are not logged in, you aready have your deployments in memory

Reply:
they come as `occupiedParcels`